### DISCUSSION TITLE
Is there any option to lock a node so that the text can't be edited and the node can't bedeleted?

### DISCUSSION THREAD

**Discussion by adamfreeplaneuser01** [freeplane/freeplane#1559] [Help]
I'm not seeing an option for that, but I think it would make a great feature because I constantly find myself slipping on the keyboard any typing over a node that I don't want to edit

**Comment by i-plasm** [comment]
There are two "modes" :  ` Edit -> Editing `  and ` Edit -> Viewing `. The last one will block any kind of editing on the map (except actions like Undo/Redo, etc). That said, the ` Viewing ` mode does not block typing on the notes panel, so if you type a note when on this mode you'll lose the note (which is the expected behaviour, but sometimes one just easily forgets that the current mode is not editable). Maybe I should suggest the devs that the note panel will be blocked as well under that mode! Hope it helps.
